content/html/concepts/semantic-html/terms/meter/meter.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: '<meter>'
+Description: 'Represents either a fractional value or a scalar value within a known range.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<meter>` semantic tag is used to represent a fractional or a scalar value within a known range. This form of representation is known as a gauge.
+
+> **Note**: The `<meter>` tag should not be used to indicate progress (as in a progress bar).
+
+## Syntax
+
+The `<meter>` is written as `<meter></meter>` with any contents inserted between the start and end tags:
+
+```html
+<meter>
+    <!-- Meter content goes here. -->
+</meter>
+```
+
```

Comment:
[Proofreading] Mention the min and max values in the syntax and a little note about how they must be implemented. Doing so would help developers and people who use coding in their work understand how to effectively use the `<meter>` element.   
```suggestion
The `<meter>` is written as `<meter></meter>` with any contents inserted between the start and end tags:

```html
<meter min="number" max="number">
    <!-- Meter content goes here. -->
</meter>
```
>**Note:** When using the max and min attribute in the `<meter>` element, they should define the range so that the value's attribute can appear within it, except if the value is between 0 and 1.
```

Reply:
You forgot to work on this one @droffilc1